qemu/tests/cfg/virtio_fs_nfs_migration.cfg

```diff
@@ -0,0 +1,172 @@
+        viofs_log_file = C:\viofs_log.txt
+        viofs_svc_name = VirtioFsSvc
+        viofs_exe_path = C:\virtiofs.exe
+        viofs_exe_copy_cmd = xcopy %s C:\ /Y
+        viofs_sc_create_cmd = 'sc create ${viofs_svc_name} binpath=${viofs_exe_path} start=auto'
+        viofs_sc_create_cmd += ' depend="WinFsp.Launcher/VirtioFsDrv" DisplayName="Virtio FS Service"'
+        viofs_sc_start_cmd = 'sc start ${viofs_svc_name}'
+        viofs_sc_query_cmd = 'sc query ${viofs_svc_name}'
+        viofs_sc_delete_cmd = 'sc delete ${viofs_svc_name}'
+        debug_log_operation = 'enable'
+        viofs_debug_enable_cmd = 'reg add HKLM\Software\VirtIO-FS /v DebugFlags /d 0xFFFFFFFF /t REG_DWORD'
+        viofs_log_enable_cmd = 'reg add HKLM\Software\VirtIO-FS /v DebugLogFile /d ${viofs_log_file} /t REG_SZ'
+        viofs_debug_delete_cmd = 'reg delete HKLM\Software\VirtIO-FS /v DebugFlags /f'
+        viofs_log_delete_cmd = 'reg delete HKLM\Software\VirtIO-FS /v DebugLogFile /f'
+        viofs_reg_query_cmd = 'reg query HKLM\Software\VirtIO-FS'
+        virtio_win_media_type = iso
+        cdroms += " virtio"
+        cmd_md5 = "%s: && md5sum.exe ${test_file}"
+        cmd_dd = 'dd if=/dev/random of=%s bs=1M count=100'
+        guest_file = "%s:\${test_file}"
+        fio_options = '--name=${fio_name} --filename=%s --ioengine=windowsaio'
+        fio_options += ' --rw=rw --bs=1k --size=200M --numjobs=${numjobs} --runtime=1800 --time_based --thread'
+        fio_name = fio.exe
+        cmd_chk_fio = 'TASKLIST /FI "IMAGENAME eq FIO.EXE'
+        fio_filename = "%s\:\${test_file}.fio"
+    variants:
+        - cache_mode_auto:
+            fs_binary_extra_options += " --cache auto"
+            fs_binary_extra_options_targetfs += " --cache auto"
+        - cache_mode_always:
+            fs_binary_extra_options += " --cache always"
+            fs_binary_extra_options_targetfs += " --cache always"
+        - cache_mode_never:
+            fs_binary_extra_options += " --cache never"
+            fs_binary_extra_options_targetfs += " --cache never"
+        - cache_mode_metadata:
+            fs_binary_extra_options += " --cache metadata"
+            fs_binary_extra_options_targetfs += " --cache metadata"
+    variants:
+        - @default:
+        - writeback:
+            only cache_mode_auto cache_mode_always
+            migrate_parameters = "{'downtime-limit': 30000}"
+            fs_binary_extra_options += " --writeback"
+            fs_binary_extra_options_targetfs += " --writeback"
+        - dio:
+            only cache_mode_never
+            fs_binary_extra_options += " --allow-direct-io"
+            fs_binary_extra_options_targetfs += " --allow-direct-io"
+    variants:
+        - find_paths:
+        - file_handles:
+            required_virtiofsd_version = [1.13.0,)
+            virtiofsd_version_cmd = rpm -q virtiofsd | cut -d- -f2
+            fs_binary_extra_options_targetfs = " --modcaps=+dac_read_search"
```

Comment:
Should be `+=` but not `=`, or the migration will fail as the cache mode does not match to source daemon.

Reply:
@zhencliu Please take this into account as well, thanks

Reply:
Sorry for the mistake, updated.